gqlgen.yml

```diff
@@ -94,3 +94,5 @@ models:
     model: github.com/evergreen-ci/evergreen/rest/model.APIUseSpruceOptions
   SiteBanner:
     model: github.com/evergreen-ci/evergreen/rest/model.APIBanner
+  EventLogEntry:
+    model: github.com/evergreen-ci/evergreen/model/event.EventLogEntry 
```

Comment:
I know it's more work (sorry!) but can you change this to APIEventLogEntry and do the needed conversions? That way the db models will stay separate from what we send to clients

Reply:
The gql type that maps to APIEventLogEntry is called TaskEventLogEntry which I realize should actually be called EventLogEntry